schema.sql

```diff
@@ -9,3 +9,35 @@ CREATE TABLE animals (
     weight_kg BIGINT NOT NULL,
 	 species VARCHAR(50)
```

Comment:
I changed the weight_kg to be numeric, and also inside the input the value I changed it to be a number instead of string.
![Screenshot from 2023-05-03 09-50-58](https://user-images.githubusercontent.com/118063058/235860603-57816ab7-caf1-4234-aa0c-d91bf4935e60.png)
![Screenshot from 2023-05-03 09-52-40](https://user-images.githubusercontent.com/118063058/235860422-77bec429-328b-49f4-9c4b-1df395847d20.png)